tests/test_msgtopdf.py

```diff
@@ -22,6 +22,19 @@ def test_init_save_path(self):
         email = Msgtopdf("C:/test/email.msg")
         assert email.save_path == pathlib.PurePath("C:/test/email")
 
+        p = pathlib.Path("C:/test/email/email")
+        p.mkdir(exist_ok=True, parents=True)
```

Comment:
Having `exists_ok=True` could overwrite existing information during a test, which is not ideal, I would leave this as the default i.e. `exists_ok=False`.